app/src/main/java/dev/sasikanth/pinnit/editor/EditorScreenModel.kt

```diff
@@ -14,8 +14,8 @@ data class EditorScreenModel(
     fun default(notificationUuid: UUID?, content: String?) = EditorScreenModel(
       notificationUuid = notificationUuid,
       notification = null,
-      title = null,
-      content = content
+      title = content,
```

Comment:
This will be a bit problematic, the reason why I was setting content here is because it can extend the limit of the title. We can pass a new param to `default` method called `title: String?` and set title and content.